### PR TITLE
Docs: Fix transparency in material browser.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: https://discourse.threejs.org/t/opacity-bug-in-materials-documentation/35251

**Description**

Since `Material.transparent` controls the `opaque` program parameter, it is necessary to call `needsUpdate` to `true` if the transparency setting changes over time.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev61/docs/scenes/material-browser.html#MeshPhongMaterial

**Comment by bergden-resonai** [comment]
I think this doc should be updated as well:
https://threejs.org/docs/?q=update#manual/en/introduction/How-to-update-things

I find it very hard to know what does or doesn't require `needsUpdate` and it's very hard to understand from reading the codebase
I'm guessing doing setters for these attributes is a bad idea because it will causes unwarranted compiles, but maybe the docs can include a mention of this in the relevant page (as well as the general "How to update things" page)?

File: docs/manual/ja/introduction/How-to-update-things.html [hunk 1]
```diff
@@ -149,6 +149,7 @@ <h2>Materials</h2>
                     <li>morphing</li>
                     <li>shadow map</li>
                     <li>alpha test</li>
+										<li>transparent</li>
```
Comment (bergden-resonai) [inline docs/manual/ja/introduction/How-to-update-things.html]: Tabs vs. spaces 😱

Comment (Mugen87) [reply docs/manual/ja/introduction/How-to-update-things.html]: This clean up can be done in a separate PR. It necessary in the entire `jp` directory.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**